File: 코딩 기초 트레이닝/마지막 두 원소-p181927/마지막 두 원소.py
def solution(num_list):
    length = len(num_list)

    # a: 마지막 전
    # b: 마지막

    a = num_list[length-2]
    b = num_list[length-1]
    num_list.append(b*2 if b <= a else b - a)

    return num_list

File: 코딩 기초 트레이닝/마지막 두 원소-p181927/마지막 두 원소.js
function solution(num_list) {
    const LEN = num_list.length;

    const A = num_list[LEN-2];
    const B = num_list[LEN-1];

    num_list.push(A >= B ? B * 2 : B - A);

    return num_list;
}
